client/src/Login.js

```diff
@@ -27,41 +31,85 @@ const Login = ({ user, login }) => {
-                Login
-              </Button>
-            </Grid>
-          </Grid>
-        </form>
+    <Grid container className={classes.root}>
+      <SplashPicture />
+      <Box className={classes.contentContainer}>
+        <Box className={classes.topContainer}>
+          <Box className={classes.topWrapper}>
+            <Typography className={classes.instructionText}>
+              Don't have an account?
+            </Typography>
+            <Button
+              href="/register"
+              to="/register"
+              color="secondary"
+              variant="contained"
+              size="large"
+            >
+              Create Account
+            </Button>
+          </Box>
+        </Box>
+        <Box className={classes.mainContainer}>
+          <Box className={classes.mainWrapper}>
+            <Typography
+              className={`${classes.headerText} ${classes.largeText}`}
+            >
+              Welcome Back!
+            </Typography>
+            <form onSubmit={handleLogin}>
+              <Grid container direction="column" spacing={3}>
+                <Grid item>
+                  <Typography className={classes.instructionText}>
+                    Username
+                  </Typography>
+                  <FormControl fullWidth margin="normal" required>
+                    <TextField
+                      aria-label="username"
+                      name="username"
+                      type="text"
+                    />
+                  </FormControl>
+                </Grid>
+                <Grid item>
+                  <Typography className={classes.instructionText}>
+                    Password
+                  </Typography>
+                  <FormControl fullWidth margin="normal" required>
+                    <TextField
+                      aria-label="password"
+                      type="password"
+                      name="password"
+                      InputProps={{
+                        endAdornment: (
+                          <InputAdornment position="end">
+                            <a
+                              className={classes.forgotLink}
+                              href="/forgot"
+                              to="/forgot"
+                            >
+                              Forgot?
+                            </a>
+                          </InputAdornment>
+                        ),
+                      }}
+                    />
+                  </FormControl>
+                </Grid>
+                <Grid container className={classes.submitContainer}>
+                  <Button
+                    type="submit"
+                    color="primary"
+                    variant="contained"
+                    size="large"
+                  >
+                    Login
+                  </Button>
+                </Grid>
+              </Grid>
+            </form>
+          </Box>
+        </Box>
```

Comment:
Hi @Fire-Hound This was because after the first issue was resolved and my pull request was approved, I pulled the new stuff from master. I'm not sure how to go about removing these files from the commit when I've already pushed these commits to the remote. Any info would be great!